src/app.ts

```diff
@@ -10,6 +10,6 @@ const productRouter = new ProductRouter()
 app.use(express.json())
 
 app.use('/demo', demoRouter.getRouter())
-app.use('/api/v1/products', productRouter.getRouter())
+app.use('/api/v1', productRouter.getRouter())
```

Comment:
How come this? I think it would be preferable to keep the `/products` here and add a `/` route in the ProductRouter

Reply:
okay will do - wasn't sure what looked more readable vs DRY